packages/playwright/src/reporter.ts

```diff
@@ -340,22 +350,41 @@ export default class ReplayPlaywrightReporter implements Reporter {
   async onEnd() {
     try {
       await this.reporter.onEnd();
-      if (!this._foundReplayBrowser) {
-        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-browser");
 
-        const output: string[] = [];
+      const output: string[] = [];
+
+      const executedProjectWithReplay = !Object.keys(this._projects).some(projectName => {
+        const { executed, usingReplay } = this._projects[projectName];
+        return executed && usingReplay;
+      });
+
+      if (!executedProjectWithReplay) {
+        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-replay-project");
         output.push(emphasize("None of the configured projects ran using Replay Chromium."));
-        if (!existsSync(getRuntimePath())) {
+      }
+
+      if (!existsSync(getRuntimePath())) {
+        mixpanelAPI.trackEvent("playwright.warning.replay-browser-not-installed");
```

Comment:
I think the only thing I would tweak about this maybe is to conditionally report this error (if `executedProjectWithReplay`) so we don't get a bunch of false positives in Mixpanel?

Reply:
good idea, i pushed out this change